src/main/java/bibliotecame/back/Extension/ExtensionController.java

```diff
@@ -1,25 +1,43 @@
 package bibliotecame.back.Extension;
 
+import bibliotecame.back.User.UserService;
 import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
 import org.springframework.http.ResponseEntity;
-import org.springframework.web.bind.annotation.PathVariable;
-import org.springframework.web.bind.annotation.PostMapping;
-import org.springframework.web.bind.annotation.RequestMapping;
-import org.springframework.web.bind.annotation.RestController;
+import org.springframework.web.bind.annotation.*;
+
+import javax.validation.Valid;
 
 @RestController
 @RequestMapping("/extension")
 public class ExtensionController {
 
     private final ExtensionService extensionService;
+    private final UserService userService;
 
     @Autowired
-    public ExtensionController(ExtensionService extensionService) {
+    public ExtensionController(ExtensionService extensionService, UserService userService) {
         this.extensionService = extensionService;
+        this.userService = userService;
     }
 
     @PostMapping("/{loanId}")
     public ResponseEntity<ExtensionModel> createExtension(@PathVariable int loanId){
         return extensionService.createExtension(loanId);
     }
+
+    @PutMapping("/{id}")
```

Comment:
Is better to have 2 endpoints to do the extension modification, because we only can change the extension status. So is better having `/extension/{ID}/approve` and `/extension/{ID}/reject`.
It makes easier to use these endpoints cause you only have to pass the ID of the extension.